docs/usage.rst

```diff
@@ -19,6 +19,16 @@ By default the import process will start by running a simple graphical editor,
 allowing you to input metadata required or recommended by the NWB format,
 but that is not available within the Labview data folder.
 
+``metadata2nwb`` creates a 'bare bones' NWB file by loading metadata describing
+an experiment from a YAML file. You need to provide it the path to the NWB file
+to create, the path to the metadata YAML file and optionally the user. If the
+metadata YAML file contains more than one user, the user must be specified. An
```

Comment:
I think it's slightly clearer to phrase this the other way around: You need to give [...] and optionally a user. The user must  be mentioned in the metadata file. If there is only one user in the file, you don't need to specify them when running this.